### ISSUE TITLE
MultiComboBox: Screenreader does not announce the readonly state

### ISSUE THREAD

**Issue by cmealares** [UI5/webcomponents-react#8935]
### Describe the bug

When MultiComboBox has the readonly property set to true, screnreaders must announce that the element is readonly.

### Isolated Example

WC4R's storybook

### Reproduction steps

1. Open WC4R storybook and go the the MultiComboBox.
2. Scroll to the first example.
3. Make a selection
4. In the properties, set readonly=true
5. Set focus on the example

Here is what JAWS reads:

> edit combo
> collapsed
> blank
> Contains 1 token
> To set the value use the Arrow keys or type the value.

### Expected Behaviour

The screenreader must announce the readonly state.
This is a requirement of SAP's product standard ACC-264.1

### UI5 Web Components for React Version

V1

### UI5 Web Components Version

V1

### Browser

Chrome

### Additional Context

It can be observed that the ComboBox component has the expected behavior.
For example, Jaws announces:

> edit combo collapsed Read Only
> ComboBox Entry 2
> To set the value use the Arrow keys or type the value.

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @cmealares 

I will close this issue as there is already [another one](https://github.com/UI5/webcomponents/issues/12705) covering this topic. If you have any questions or more information, please address them in the original issue.
